crowndata-evaluation/crowndata_evaluation/services/action_consistency/clustering.py

```diff
@@ -47,51 +48,122 @@ def sklearn_cluster_wrapper(
     return clusters
 
 
-def define_clusters(data: np.ndarray, epsilon: float) -> Dict[int, np.ndarray]:
-    r"""
-    Define clusters for a set of states or actions based on epsilon distance.
+# # TODO: add a name to the function
+# def define_clusters(data: np.ndarray, epsilon: float) -> Dict[int, np.ndarray]:
+#     r"""
+#     Define clusters for a set of states or actions based on epsilon distance.
+
+#     For each state/action ``x_i``, we define a cluster ``C(x_i)`` as:
+
+#     .. math::
+#         C(x_i) = \{ x_j | \|x_i - x_j\| \leq \epsilon \}
+
+#     where :math:`\|x_i - x_j\|` is the Euclidean distance between state/action
+#     ``x_i`` and ``x_j``.
+
+#     Parameters
+#     ----------
+#     data : np.ndarray
+#         State or action data to cluster. (x, y, z, rotation_x, rotation_y, rotation_z)
+#     epsilon : float
+#         The coverage distance for clustering.
+
+#     Returns
+#     -------
+#     Dict[int, np.ndarray]
+#         A dictionary where each key is a cluster index, and the value is the
+#         states or actions in that cluster.
+#     """
+
+#     clusters = {}
```

Comment:
these code are obsolete, can be removed.

Reply:
> these code are obsolete, can be removed.

https://github.com/crowndata/crowndata-api-python/pull/11/commits/e338c025ed72efe72b216120d9de189608f5c1f7

- code clean
- scikit-learn style comment